tests/deployment/test_client_server.py

```diff
@@ -106,23 +107,18 @@ def test_client_server_sklearn(
     with pytest.raises(AttributeError, match=".* model is not compiled.*"):
         client_server_simulation(x_train, x_test, model, default_configuration)
 
-    fhe_circuit = model.compile(
-        x_train, default_configuration, **extra_params, show_mlir=(n_bits <= 8)
-    )
-
-    if get_model_name(model) == "KNeighborsClassifier":
-        # Fit the model
-        with warnings.catch_warnings():
-            # Sometimes, we miss convergence, which is not a problem for our test
-            warnings.simplefilter("ignore", category=ConvergenceWarning)
-            model.fit(x, y)
+    fhe_circuit = model.compile(x_train, default_configuration, **extra_params, show_mlir=False)
 
     max_bit_width = fhe_circuit.graph.maximum_integer_bit_width()
     print(f"Max width {max_bit_width}")
 
     # Check that the FHE execution is correct.
     # With a global_p_error of 1/100_000 we only allow one run.
```

Comment:
should these two comments be removed then ?

Reply:
Are you talking about:
# Check that the FHE execution is correct.
# With a global_p_error of 1/100_000 we only allow one run.

Reply:
yes I am, since you added comments below, or maybe merge them together ?

Reply:
I'll merge them